app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -219,6 +219,17 @@ class Meta:
         model = NimbusExperiment
         fields = ("probe_sets",)
 
+    def update(self, experiment, data):
+        probe_sets = data.pop("probe_sets")
+        experiment = super().update(experiment, data)
+        with transaction.atomic():
+            for probe_set in probe_sets:
+                experiment.probe_sets.add(
+                    probe_set, through_defaults={"is_primary": True}
```

Comment:
So we'll setup the primary/secondary distinction in the other pr?

Reply:
Yup.